Firebase/Auth/Source/FIRAuthErrorUtils.h

```diff
@@ -460,12 +460,27 @@ NS_ASSUME_NONNULL_BEGIN
 + (NSError *)webContextAlreadyPresentedErrorWithMessage:(nullable NSString *)message;
 
 /** @fn webContextCancelledErrorWithMessage:
-    @brief Constructs an @c NSError with the @c FIRAuthErrorCodeWebContextCancelledcode.
+    @brief Constructs an @c NSError with the @c FIRAuthErrorCodeWebContextCancelled code.
     @param message Error message from the backend, if any.
     @return The NSError instance associated with the given FIRAuthError.
  */
 + (NSError *)webContextCancelledErrorWithMessage:(nullable NSString *)message;
 
+/** @fn appVerificationUserInteractionFailureWithReason:
+    @brief Constructs an @c NSError with the @c
+        FIRAuthErrorCodeAppVerificationUserInteractionFailure code.
+    @param reason Reason for error, returned via URL response.
+    @return The NSError instance associated with the given FIRAuthError.
+ */
++ (NSError *)appVerificationUserInteractionFailureWithReason:(nullable NSString *)reason;
+
+/** @fn reCAPTCHAResponseErrorWithCode:message:
+    @brief Constructs an @c NSError with the code and message provided.
+    @param message Error message from the backend, if any.
+    @return The NSError instance associated with the given FIRAuthError.
```

Comment:
@return The NSError instance associated with the given error message, if one is found.

Reply:
done.